politeiawww/api/v1/v1.go

```diff
@@ -930,7 +930,8 @@ type GetComments struct {
 
 // GetCommentsReply returns the provided number of comments.
 type GetCommentsReply struct {
-	Comments []Comment `json:"comments"` // Comments
+	Comments   []Comment `json:"comments"`   // Comments
+	AccessTime int64     `json:"accesstime"` // User Access Time
```

Comment:
This should be optional and the doco should reflect that. When not logged in we don't return it.